plugins/protocolbuffers/pyi/v30.2/buf.plugin.yaml

```diff
@@ -14,5 +14,5 @@ registry:
     requires_python: ">=3.9"
     deps:
       # https://pypi.org/project/protobuf/
-      - "protobuf~=6.30.2"
-      - "types-protobuf~=5.29"
+      - "protobuf>3.20"
```

Comment:
Should this be >=?

Reply:
https://github.com/bufbuild/plugins/pull/2053/commits/6e1b5e7c50fc945fb70b2927c62788dd0b28f4c8